src/app/plugins/scores.plugin.ts

```diff
@@ -0,0 +1,125 @@
+import { RichEmbed } from 'discord.js';
+import { Plugin } from '../../common/plugin';
+import Constants from '../../common/constants';
+import { IContainer, IMessage, ChannelType } from '../../common/types';
+
+
+export class ScoresPlugin extends Plugin {
+  public name: string = 'NCAA Scores Plugin';
+  public description: string = 'Gets score of a football game.';
+  public usage: string = 'scores <team name>; ex scores UCF';
+  public permission: ChannelType = ChannelType.Public;
+  public pluginChannelName: string = Constants.Channels.Public.Sports;
+
+  constructor(public container: IContainer) {
+    super();
+  }
+
+  public async execute(message: IMessage, args?: string[]) {
+    const endpoint = 'http://www.espn.com/college-football/bottomline/scores';
+    let desiredTeamQuery = this._parseTeam(args || []);
+
+    //If no args, team is UCF by default
+    if (desiredTeamQuery === '') {
+      desiredTeamQuery = 'UCF';
+    }
+
+
+    try {
+      const response = await this.container.httpService.get(endpoint);
+      const games = response.data.split('?'); //Each game ends with a ?
+
+
+      let teamFound = false;
+      for (const game of games) {
+        if (this._containsAllTokens(game, desiredTeamQuery.split(' '))) {
+          teamFound = true;
+
+          const visitorDataRegex = '[=\\\^][a-zA-Z0(%20)]+%20[0-9]+'; //Isolates visiting team's data
+          const homeDataRegex = '%20%20\\\^?(\\\(\\\d+\\\))?[a-zA-Z(%20)]+%20[0-9]+'; //Isolates home team's data
```

Comment:
Move the regex's out of the loop because they're common

nit: Prefer the `/regex/` syntax so you don't have to escape all your backslashes.